nss-day-cohort-30/bangazon-api-maniacal-magpies
# Training program es

# Description
Make sure to add new data to the database BEFORE you run tests! Training program module and associated tests are fully functional. 

## Type of change

# Testing Instructions for Change Made
1. `git fetch --all`
1. `git checkout training-program-ES`
1. `insert SQL below into BangazonAPI database using Azure Data Studio`
``` 
INSERT INTO TrainingProgram ([Name], StartDate, EndDate, MaxAttendees)
VALUES ('Cyber Awareness', '05-20-2019', '05-21-2019', '20');
INSERT INTO TrainingProgram ([Name], StartDate, EndDate, MaxAttendees)
VALUES ('Windows for Mac Users', '06-01-2019', '07-03-2019', '10');
INSERT INTO TrainingProgram ([Name], StartDate, EndDate, MaxAttendees)
VALUES ('Personal Hygiene for Programmers', '08-10-2019', '08-12-2019', '15');
INSERT INTO TrainingProgram ([Name], StartDate, EndDate, MaxAttendees)
VALUES ('Equity: As good as money!', '02-25-2019', '02-27-2019', '10');

INSERT INTO EmployeeTraining (EmployeeId, TrainingProgramId)
VALUES (1, 1);
INSERT INTO EmployeeTraining (EmployeeId, TrainingProgramId)
VALUES (1, 2);
INSERT INTO EmployeeTraining (EmployeeId, TrainingProgramId)
VALUES (1, 3);
INSERT INTO EmployeeTraining (EmployeeId, TrainingProgramId)
VALUES (2, 1);
INSERT INTO EmployeeTraining (EmployeeId, TrainingProgramId)
VALUES (2, 2);
INSERT INTO EmployeeTraining (EmployeeId, TrainingProgramId)
VALUES (3, 1);
INSERT INTO EmployeeTraining (EmployeeId, TrainingProgramId)
VALUES (3, 1); 
```
4. `Run TestTrainingPrograms tests and ensure that all six tests pass.`
5. `View code in TrainingProgramController.cs to ensure that it follows project conventions`


# Checklist:
